static/ui.js

```diff
@@ -17,7 +17,7 @@ function createChildDependeciesListItems(queryPackage, parentID) {
   if (parent.children.length > 0) {
     return;
   }
-  fetch("http://localhost:5000/" + queryPackage)
+  fetch("http://127.0.0.1:5000/" + queryPackage)
```

Comment:
dont use any base url here.\
your browser can detect it if you just start your path from `/`:
```js
fetch("/" + queryPackage)
```

Reply:
amazing!